hardware/plugins/Plugins.cpp

```diff
@@ -240,7 +240,7 @@ namespace Plugins {
 				if (type == 1) type = PDM_ALL;
 
 				pModState->pPlugin->m_bDebug = (PluginDebugMask)type;
-				_log.Log(LOG_NORM, "(%s) Debug logging mask set to: %s%s%s%s%s%s%s%s%s", pModState->pPlugin->Name.c_str(), 
+				_log.Log(LOG_NORM, "(%s) Debug logging mask set to: %s%s%s%s%s%s%s%s%s", pModState->pPlugin->Name.c_str(),
```

Comment:
could you add this in a #define _DEBUG block so this is only printed when compiled in debug mode ?

Reply:
@dnpwwo very recently did big changes to plugin framework in bf379c441ff72085ebe51b32d9008546acc1cefa to have much more fine grained control over prints, I think the purpose of this print is to make the plugin developer and user know that the prints are in backwards compatible mode. It would be better if @dnpwwo removes/hides it once he's sure everyone has adjusted to the new print control..